config/printer-eryone-thinker-series-v2.cfg

```diff
@@ -0,0 +1,225 @@
+# This file contains pin mappings for the Eryone Thinker Series V2.
+# To use this config, the firmware should be compiled for the AVR atmega2560.
+# There are settings to support BL Touch and Filament Runout sensor,
+# which are commented out by default as the print does ship with the support
+# but the user needs purchase them seperately
+
+# Author: dapostol73
+# Thanks to 3D.Subzwari and MAD3D for the help
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC0
+dir_pin: !PL1
+enable_pin: !PA7
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PB6
+position_min: -5
+position_endstop: -5
+position_max: 300
+homing_speed: 50
+
+[stepper_y]
+step_pin: PC1
+dir_pin: PL0
+enable_pin: !PA6
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PB5
+position_min: -5
+position_endstop: -5
+position_max: 300
+homing_speed: 50
+
+[stepper_z]
+step_pin: PC2
+dir_pin: PL2
+enable_pin: !PA5
+microsteps: 16
+rotation_distance: 8
+position_max: 400
+homing_speed: 25
+# Stock setting for endstop, remove these lines for BL Touch
+endstop_pin: !PB4
+position_endstop: 0
+# Bl Touch Uncomment the following lines and comment out Stock
+#endstop_pin: probe:z_virtual_endstop
+#position_min: -2
+
+[extruder]
+step_pin: PC3
+dir_pin: !PL6
+enable_pin: !PA4
+microsteps: 16
+rotation_distance: 32
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PH6
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PF0
+control: pid
+pid_kp: 22.2
+pid_ki: 1.08
+pid_kd: 144
+min_temp: 0
+max_temp: 250
+max_extrude_only_distance: 300
+# Calibrate - see https://github.com/KevinOConnor/klipper/blob/master/docs/Pressure_Advance.md
+#pressure_advance: 0.0
+
+[fan]
+pin: PH5
+
+[heater_fan nozzle_cooling_fan]
+pin: PH3
+
+[heater_bed]
+heater_pin: PE5
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PF2
+control: pid
+pid_kp: 100.0
+pid_ki: 5.023
+pid_kd: 305.4
+min_temp: 0
+max_temp: 100
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 1500
+max_z_velocity: 5
+max_z_accel: 150
+
+[pause_resume]
+
+[virtual_sdcard]
+path: ~/.octoprint/uploads/
+
+[respond]
+
+######################################################################
+##  Filament runout sensor settings
+##  Uncomment the following if using BL Touch and adding runout sensor
+######################################################################
+
+#[filament_switch_sensor filament_runout_sensor]
+#runout_gcode: M600
+#switch_pin: !PB4
+
+[gcode_macro M600]
+gcode:
+    PAUSE
+    CHANGE_FILAMENT_START
+
+[gcode_macro CHANGE_FILAMENT_START]
+gcode:
+    SAVE_GCODE_STATE NAME=M600_state
+    G91
+    G1 E-.8 F2700
+    G1 Z10
+    G90
+    G1 X50 Y0 F3000
+    G91
+    G1 E-5 F300
+
+[gcode_macro CHANGE_FILAMENT_END]
+gcode:
+    G91
+    RESTORE_GCODE_STATE NAME=M600_state
+    RESUME
```

Comment:
Similarly, unless I'm missing something, these macros don't look like they are specific to the printer.

Reply:
No they are not, but their is feedback from the community that they would like these to be easier for none technical people to not have figure this out on their own....is it cool to have these commented out, similar to the BL Touch settings?

Reply:
I'll remove them and add example to their Facebook Group